packages/xo-web/src/common/intl/messages.js

```diff
@@ -571,6 +571,7 @@ const messages = {
     'Delete backup job{nJobs, plural, one {} other {s}}',
   confirmDeleteBackupJobsBody:
     'Are you sure you want to delete {nJobs, number} backup job{nJobs, plural, one {} other {s}}?',
+  runBackupJob: 'Run backup job',
```

Comment:
Maybe "Run backup job once"? So that it's very clear that it has nothing to do with *enabling* the scheduled job.